gst/droidcodec/gstdroidvdec.c

```diff
@@ -908,6 +908,19 @@ gst_droidvdec_configure_state (GstVideoDecoder * decoder, guint width,
   return FALSE;
 }
 
+static inline void
+codec_destroy_fn (DroidMediaCodec * codec)
```

Comment:
I find this function name a bit strange. What is `_fn` suffix supposed to mean?

Reply:
It was a thread function

Reply:
but not any more